rest/utilities_testing.go

```diff
@@ -648,24 +674,26 @@ func (bt BlipTester) Close() {
 }
 
 // Create a BlipTester using the default spec
-func NewBlipTester() (*BlipTester, error) {
-	defaultSpec := BlipTesterSpec{}
-	return NewBlipTesterFromSpec(defaultSpec)
+func NewBlipTester(tester testing.TB) (*BlipTester, error) {
+	var rt = NewRestTester(tester, nil)
+	defaultSpec := BlipTesterSpec{restTester: rt}
+	return NewBlipTesterFromSpec(tester, defaultSpec)
```

Comment:
If we've changed `NewBlipTesterFromSpec` to take a tester explicitly, do we still need to create a new RestTester here just for defaultSpec?